docs/swagger.json

```diff
@@ -1681,21 +1681,40 @@
         "operationId": "GET LoRaWAN Concentrator list",
         "parameters": [
           {
-            "name": "pageSize",
+            "name": "keyword",
```

Comment:
@hocinehacherouf @GuillaumeM-2ISA , in my mind other parameters for the search are named ``ìd`` in that case It should be more convenient to keep the same name here ?